src/config/index.js

```diff
@@ -213,6 +215,22 @@ let config = {
   components,
 };
 
+// The apiExpanders depends on a config of the object, so it's done here
+config.settings.apiExpanders = [
+  ...config.settings.apiExpanders,
+  {
+    match: '',
+    GET_CONTENT: ['breadcrumbs', 'actions', 'types'],
+  },
+  {
+    match: '',
+    GET_CONTENT: ['navigation'],
+    querystring: {
+      'expand.navigation.depth': config.settings.navDepth,
```

Comment:
Does this work if final projects change `navDepth`? I am not entirely sure, or I don't get why it works at the moment.

Reply:
You are correct, it won't... since it's passed once, with the "old", default value... Not an easy one :/

Suggestions?

Reply:
@sneridagh An idea: make it possible to pass a function that constructs the querystring, instead of a prebuilt querystring object.

Reply:
@davisagli see https://github.com/plone/volto/pull/4946/commits/3bcffd6ef10b98754149f4f5b2651606a379d4ec

Reply:
@pnicolli it turns out we can't because we haven't set `config.settings.navDepth`, unless we assume we are going to sync them manually (I guess that the default is not going to change that much).